hub/scripts.js

```diff
@@ -245,6 +245,26 @@
     });
   }
 
+  /**
+   * Loads a JS file on the page
+   * @param {String} src The path to the JS file
+   * @param {Object} param The parameters used to load the file
+   */
+  function loadJS(src, param) {
+    if (isNonEmptyString(src)) {
+      const script = document.createElement('script');
+      script.setAttribute('src', src);
+      script.setAttribute('type', 'text/javascript');
+      if (typeof param === 'object') {
+        if (isNonEmptyString(param.id)) {
```

Comment:
```suggestion
      if (typeof param === 'object' && isNonEmptyString(param.id)) {
```

Reply:
I thought we might enhance this method to pass along other attributes such as 'module', 'defer'. I can switch to using `id` for now since that's used.